docs/changelog.rst

```diff
@@ -10,6 +10,7 @@ Changelog
 * :bug:`5128` Ethereum transactions where no value was transferred will now be correctly decoded.
 * :bug:`-` Uniswap V3 oracle will now correctly skip assets with no liquidity when querying prices.
 * :bug:`2934` Users will now be able to deploy the docker image on a subpath instead of the root domain.
+* :bug:`-` Users won't get errors querying prices due the usage of defi oracles for querying prices of NFTs.
```

Comment:
We haven't removed all the errors from price queries because we can't be certain of that. What we did was fix an error when an NFT was queried using the uniswap oracle. If you check that address is an NFT but the proble is that the uniswap oracle was assuming that every token queried was an ERC20 and was building the identifiers for that